doc/articles/contributing/guidelines/updating-dependencies.md

```diff
@@ -0,0 +1,57 @@
+# Guidelines for updating dependencies
+
+We use Dependabot to notify the team of any updates to dependencies. Once a week the robot will scan our dependencies and raise a pull-request if a new version is found. If an existing open pull-request is found for a dependency it will be closed and replaced with a new pull-request. The behaviour of the robot is [controlled by this configuration file](https://github.com/unoplatform/Uno/blob/master/.dependabot/config.yml). 
+
+## internal dependencies
+
+These dependencies don't change the public API surface and are typically safe to merge and we could potentially [configure mergify to automatically merge them if CI passes](https://medium.com/mergify/merging-bots-pull-requests-automatically-548ed0b4a424):
+
+- [CommonServiceLocator](https://github.com/unoplatform/uno/pull/1174)
+- BenchmarkDotNet
+- FluentAssertions 
+- NUnit3TestAdapter 
+- [NUnit.Runners](https://github.com/unoplatform/uno/pull/1122)
+- [Microsoft.AppCenter](https://github.com/unoplatform/uno/pull/1175)
+- [MSTest.TestAdapter](https://github.com/unoplatform/uno/pull/1126)
+- [MSTest.TestFramework](https://github.com/unoplatform/uno/pull/1128)
+- Moq
+- [Xamarin.Build.Download](https://github.com/unoplatform/uno/pull/1123)
+
+These dependencies require manual adjustments before merging:
+
+- [docfx.console](https://github.com/unoplatform/Uno/pull/1082/commits/c222caf8c23b35e19f6b33cd624cbfa714250bfe)
+
+
+## public dependencies
+
+Updating these dependencies will require consumers to upgrade their dependencies and as such need consideration on a case by case basis is required before merging:
+
+- [System.Reactive](https://github.com/unoplatform/uno/pull/1170#pullrequestreview-256670600)
```

Comment:
This one (`System.Reactive`) is only used in the WpfHost which would eventually be deprecated.